var/spack/repos/builtin/packages/libxsmm/package.py

```diff
@@ -82,6 +82,9 @@ class Libxsmm(MakefilePackage):
     # (<https://github.com/spack/spack/pull/21671#issuecomment-779882282>).
     depends_on("binutils+ld+gas@2.33:", type="build", when="@:1.17")
 
+    # Intel Architecture or compatible CPU required
+    requires("target=x86_64:")
```

Comment:
@hfp is this OK? I wanna turn the configure/build time error into a constraint, s.t. packages like dbcsr automatically use blas instead of libxsmm on arm / powerpc